src/batou/lib/supervisor.py

```diff
@@ -234,7 +234,21 @@ def configure(self):
                 command=self.expand('{{component.workdir}}/check_supervisor'))
 
 
-class RunningSupervisor(Component):
+class RunningHelper(Component):
+
+    def is_running(self):
+        try:
+            pid, err = self.cmd('bin/supervisorctl pid')
+            try:
+                int(pid) > 0
```

Comment:
I know that you've just copied this over but there's been some weird cargo cult copying over the years and I can't make any sense of the int(pid) > 0 thing... Why do we compare to 0? What should happen if its 0? I think we might just drop the comparison... but I'm going to merge the PR anyway.